testsuites/syncgateway/functional/topology_specific_tests/multiple_accels/test_multiple_accels.py

```diff
@@ -136,7 +136,7 @@ def test_dcp_reshard_sync_gateway_comes_up(params_from_base_test_setup, sg_conf)
             log_info("{} Completed:".format(tag))
 
     # TODO better way to do this
-    time.sleep(60)
+    time.sleep(120)
```

Comment:
No this is related to an intermittent test failure where changes are still processing from time to time.